packages/create-mechanic/function-templates/d3-image/index.js

```diff
@@ -0,0 +1,82 @@
+import * as d3 from "d3";
+
+export const handler = ({ inputs, mechanic }) => {
+  const { width, height, text, color1, color2, radiusPercentage } = inputs;
+
+  const center = [width / 2, height / 2];
+  const radius = ((height / 2) * radiusPercentage) / 100;
+  const angle = Math.random() * 360;
+
+  let SVG = d3.create("svg")
+  .attr("width", width).attr("height", height)
+
+  let arc1 = d3.arc().innerRadius(0).outerRadius(radius)
+  .startAngle(0).endAngle(2*Math.PI/2);
+
+  let arc2 = d3.arc().innerRadius(0).outerRadius(radius)
+  .startAngle(2*Math.PI/2).endAngle(4*Math.PI/2);
```

Comment:
Shouldn't these use `const` ? Is there something I'm missing where `let` is necessary?

Reply:
`const`s everywhere! nice!